spec/display_board_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'stringio'
+
+def display_board(console, board)
```

Comment:
Try putting this in a separate file in `lib/`, like with the `find_winner` function you made before.